os/drivers/wireless/realtek/wifi_util_interface/rtk_drv.c

```diff
@@ -31,6 +31,9 @@
 #include <errno.h>
 #include <net/if.h>
 #include <tinyara/lwnl/lwnl.h>
+#ifdef CONFIG_NET_NETMGR
+#include <tinyara/netmgr/netdev_mgr.h>
+#endif
```

Comment:
did it fix the below issue as well? config QEMU build.
https://github.com/Samsung/TizenRT/issues/4759

Reply:
NO.